### ISSUE TITLE
ggeffect for MCMCglmm

### ISSUE THREAD

**Issue by mastoffel** [strengejacke/ggeffects#130]
Hallo,

first of all: great package! I wondered about the support for MCMCglmm, as it doesn't seem to work with logit or probit regressions (not sure about others):

```
library(MCMCglmm)
library(ggeffects)
intercept <- -6.0
b <- 2.75
x <- rnorm(100, 1, 3)
linpred <- intercept + x * b
prob1 <- exp(linpred) / (1 + exp(linpred))
run1 <- runif(100, 0, 1)
groups <- sample(c(1,2,3), size = 100, replace = TRUE)
y <- ifelse(run1 < prob1, 1, 0)
df <- data.frame(y = y, x = x, gr = groups)
mod <- MCMCglmm(y ~ x, random=~gr, 
                data = df, family = "categorical",
                verbose = F, nitt = 2000, burnin = 500, thin = 1)

ggeffect(mod, terms = "x" ) 
ggpredict(mod, terms = "x")
ggemmeans(mod, terms = "x")
```

Here is a screenshot:
![Screenshot 2020-04-03 at 16 21 52](https://user-images.githubusercontent.com/7348440/78377362-6ca64880-75c7-11ea-96cb-456ceed3d261.png)

Thanks!

Martin

**Comment by strengejacke** [comment]
MCMCglmm are a bit more difficult, due to its flexibility. I have now made at least `ggemmeans()` working with your example, you need to update [*insight*](https://github.com/easystats/insight/) and _ggeffects_ from GitHub to test.

**Comment by mastoffel** [comment]
Great, thanks  a lot for the quick response. It works now, but I think (if I did everything correctly) that the predicted values are not on the response scale yet, despite back.transform = TRUE.

**Comment by strengejacke** [comment]
Yes, MCMCglmm hides the link-inverse function quite well, must dig a bit more into the details.
